Makefile.am

```diff
@@ -193,6 +193,7 @@ endif
 
 
 collectd_SOURCES = \
+	src/daemon/collectd_linux.c \
```

Comment:
The code in this file isn't actually Linux specific but for all the (non-Windows) platforms currently supported by collectd. Since this file continues to include main(). I suggest to continue to call it collectd.c and special cases like Windows could then be collectd_windows.c. The code that you left in collectd.c / collectd.h would move to files like cmd.c / cmd.h and should only be used / linked into the main program (to avoid leaking symbols into other places).